### PR TITLE
Project Overview page

### LINKED ISSUE #8: Final task : Project
Design: https://www.figma.com/file/8tM7ECvw3FtoSfsHYCpfBr/Learn-with-Ajay-0-Personal-Website?node-id=0%3A1

### PR DESCRIPTION
**Pull request by Divya-Paul**
closes #8 
Completed Project

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -6,21 +6,26 @@
   --primary-color: #7562e0;
   --secondary-color: #a696ff;
   --parent-color: #120644;
+  --Tile-color: #f5f5f6;
```
Comment (ajayesivan) [inline styles/styles.css:9]: use only small letters for names

File: styles/styles.css [hunk 2]
```diff
@@ -121,3 +136,293 @@ body {
   right: -386px;
   top: 242px;
 }
+
+.Overview-container {
+  height: 120vh;
+  width: 100%;
+  position: relative;
+  /* background-color: blue; */
```
Comment (ajayesivan) [inline styles/styles.css:144]: Remove commented code

**Review by ajayesivan — APPROVED** [review]

File: styles/styles.css (hunk 1 above)
Comment (Divya-Paul) [reply styles/styles.css:9]: Okay

File: styles/styles.css (hunk 2 above)
Comment (Divya-Paul) [reply styles/styles.css:144]: Sure

**Pull request merged**